### PR TITLE
Updated instructions and screenshots for Mac / RC2

### PR DESCRIPTION
**Pull request by ardalis**

### CODE REVIEW

**Comment by BrennanConroy** [comment]
Warnings probably aren't the best thing to have in a picture in the docs https://github.com/aspnet/Docs/blob/f8c740984fec81fdf7a605d12b5d64d1f92b02e2/aspnet/client-side/yeoman/_static/dotnet-build-run.png, should the generator be updated so it doesn't produce warnings?

File: aspnet/client-side/yeoman.rst [hunk 1]
```diff
@@ -30,37 +30,47 @@ Install Node.js, npm, and Yeoman
 
 -  The ``–g`` flag installs the generator globally, so that it can be used from any path.
 
-Create an ASP.NET app
--------------------------
+Create an ASP.NET App
```
Comment (Rick-Anderson) [inline aspnet/client-side/yeoman.rst]: modern style is not to Cap, so leave it as app

Comment (Rick-Anderson) [inline aspnet/client-side/yeoman.rst]: @BrennanConroy I got similar warning (on Windows 10) when I didn't have the final RC2 bits. The warnings might go away with final RC2 and we can reshoot the image.

**Comment by danroth27** [comment]
Handling this as part of https://github.com/aspnet/Docs/pull/1228

**Pull request closed without merging**